Dockerfile

```diff
@@ -15,6 +15,11 @@ FROM singlecellportal/phusion_passenger-ruby25:1.0.6
 #FROM singlecellportal/phusion_passenger-nodejs:<VERSION>
 #FROM singlecellportal/phusion_passenger-customizable:<VERSION>
 
+ARG VCS_REF="unspecified"
+LABEL org.label-schema.vendor="Broad Institute" \
+      org.label-schema.vcs-url="https://github.com/broadinstitute/scp-rails-baseimage" \
+      org.label-schema.vcs-ref="$VCS_REF"
+
```

Comment:
Totally not a blocking concern, but it might be nice to have the version tags of the sub-images in here as well if possible.

Reply:
hopefully the latest commit addresses this for you, at least MVP-level.